Cargo.toml

```diff
@@ -3,14 +3,23 @@ name = "quickstitch"
 version = "0.1.0"
 edition = "2021"
 
+[features]
+gui = ["dep:iced"]
+
 [dependencies]
 anyhow = "1.0.86"
-image = "0.25.2"
+image = { version = "0.25.2", features = ["rayon", "jpeg", "png", "webp"] }
```

Comment:
I don't actually know what rayon support is vital for, since I'm not familiar with the library. Included it anyway just in case.